bot/constants/env_variables.py

```diff
@@ -1,28 +1,39 @@
 import os
 
-from constants.logger import logger
 import re
 
 
 def read_list_from_env(name, required_type):
     return list(
         map(lambda element: required_type(element),
-            filter(lambda element: element, re.split(",|;", os.getenv(name, '')))))
+            filter(lambda element: element, re.split("[,;]", os.getenv(name, '')))))
+
+
+def get_lcd_url(network_mode: str, debug: bool) -> str:
+    if 'LCD_ENDPOINT' in os.environ:
+        return os.environ['LCD_ENDPOINT']
```

Comment:
I specified `LCD_ENDPOINT` in the variables file, but left it empty.
Then I got 
```
requests.exceptions.MissingSchema: Invalid URL 'gov/proposals': No schema supplied. Perhaps you meant http://gov/proposals?
```

Reply:
also need to check if `LCD_ENDPOINT != ""` or something